### DISCUSSION TITLE
WinEdt 11.2 [Build: 20240317] beta

### DISCUSSION THREAD

**Discussion by Alex-Simonic** [WinEdt-Team/WinEdt-Support#22] [2.1 Announcements]
A beta version of WinEdt 11.2 [Build: 20240317] is now available for (early bird) beta testing. Follow the link:

  [https://www.winedt.com/beta.html](https://www.winedt.com/beta.html)

and carefully read the instructions. You can take your time and explore the new settings before (manually) importing your WinEdt 11.1 local configuration.

Comments, questions, problems, suggestions?

You can start a discussion on this forum...

**Comment by Alex-Simonic** [comment]
Based on the feedback, or should I say lack thereof, there isn't much interest in the portable beta version. Perhaps a missed learning opportunity.

Anyway, a proper beta version with an installer is now available. Let's see.

https://www.winedt.com/beta.html

**Comment by tozGitHub** [comment]
Dear Alex,

Is there sort of a deadline until when you would like us to give feedback? Personally, I simply did not have time yet to play around with the new version, since I have some commitments that I have to take to care of. I am certainly looking forward to a less busy time to look into the new version.

Best wishes,
Thomas

**Comment by RaghuNaik** [comment]
I installed the portable version yesterday. The installation went through flawlessly.

**Comment by urmel1960** [comment]
I installed the portable beta version as soon as it was available, and I think it's a good idea to be able to use the beta version in parallel with the stable non-beta version.

Unfortunately, there are other things to do in this world besides the beta test - so it's not always possible to give the beta test top priority. Perhaps others feel the same way. 

I think it's a bit premature to draw the conclusion that nobody is interested in it.

Thanks again for making the beta version available - especially the portable version. Unfortunately, with the update of the beta build, the introductory information on the correct handling of the portable beta has been lost. 

Is there a chance to persistently store these instructions somewhere and get them again?
What are the correct steps to update the portable beta to the latest beta build ?

Thanks and best regards

**Reply by Alex-Simonic** [reply]
I also thought that a portable beta version was a good idea. But I never know... It may not be for everyone.
The instructions are back and so is winedt11.zip (portable):

https://www.winedt.com/beta.html

By all means, take your time. WinEdt is just a tool. Your work should always be a priority.

**Reply by tozGitHub** [reply]
Dear Alex, dear community,

I beg to differ. WinEdt is much more than just a tool. I just realized it again these days when we were working on a grant application (we discovered the call too late, and had to surrender, so I hope I now have time to play a bit with WinEdt) using an online tool. It is great, but there were quite a few moments when I thought that if it were WinEdt, I would just think what to do, and it would read my mind and simply do it.
I also use WinEdt sometimes for tasks that I think Windows should provide. When I had to switch from Windows 8.1 to 11, my beloved start menu was gone (gone is too dramatic: the new one just is not the way I would like it to be). Hence, I use now simply a WinEdt instance:
![ee082d49aac8efa758802586cc43cd9d_MD5](https://github.com/WinEdt-Team/WinEdt-Support/assets/52357589/634a3e9d-06fb-408d-8c2b-454188e69e38)

For people wondering: the help explains under WinEdt Macro Manual | Command Line Switches | Example 3 how to use different configurations. What you see is just basically everything removed from WinEdt except the TeX-Symbols-panel. The idea was probably to have the tiles in black and white and simple use them to insert characters, but as we see, this does not have to be the case. We can execute arbitrary command via the buttons.

I sometimes also use WinEdt when I am looking for files. I do not see how to search with the help of regular expressions (maybe now it would be easier possible as there is some Linux-functionality added to Windows), so in such a case, I simply use WinEdt.

To add one more thing, I also think that the portable version is a great idea.

Best,
Thomas
